### PR TITLE
#620: Fix setChildren overwriting the executor

### PR DESCRIPTION
**Pull request by Xemiru**
Address the issue mentioned in #620.

### CODE REVIEW

File: src/main/java/org/spongepowered/api/util/command/spec/CommandSpec.java [hunk 1]
```diff
@@ -210,8 +207,18 @@ public Builder setInputTokenizer(InputTokenizer parser) {
          * @return the new spec
          */
         public CommandSpec build() {
-            Preconditions.checkNotNull(this.executor, "An executor is required");
-
+            if(childCommandMap == null) {
```
Comment (zml2008) [inline src/main/java/org/spongepowered/api/util/command/spec/CommandSpec.java]: this is wrong

Comment (Xemiru) [inline src/main/java/org/spongepowered/api/util/command/spec/CommandSpec.java]: didn't even see what you said

File: src/main/java/org/spongepowered/api/util/command/spec/CommandSpec.java [hunk 2]
```diff
@@ -30,6 +30,7 @@
 import com.google.common.base.Optional;
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableList;
+
```
Comment (boformer) [inline src/main/java/org/spongepowered/api/util/command/spec/CommandSpec.java]: Remove this blank line

**Comment by boformer** [comment]
Looks good to me.

**Comment by KieronWiltshire** [comment]
Why has this not been merged yet?

**Comment by Zidane** [comment]
@KieronWiltshire 

Things get merged when review has been done appropriately. People have lives outside Sponge.

**Pull request closed without merging**